docs/setup-guide/edge_cluster_copy.md

```diff
@@ -0,0 +1,91 @@
+#	File or Folder Copy Process Between Edge cluster(For Reference)
+
+## Abstract
+The purpose of this document is to how to copy file or folder in machine A to machine B and C Virtual Machine(or Cluster), and describe the each step to create copy process and refence documentation.
```

Comment:
"is to how to" gramma doesn't look right

Reply:
Updated